apps/src/lib/levelbuilder/lesson-editor/ObjectivesEditor.jsx

```diff
@@ -51,6 +52,99 @@ const styles = {
   }
 };
 
+const ObjectiveLine = onClickOutside(
```

Comment:
Can you put ObjectiveLine in its own file so we keep the react files small and easy to look at?

Reply:
Done!

Reply:
Thank you!